docs/versioned_docs/version-v0.5/guides/setting-up-password-hashing-parameters.md

```diff
@@ -24,7 +24,7 @@ $ kratos hashers argon2 calibrate 1s
 It will output the exact values to use in the
 [configuration](../reference/configuration.md).
 
-Head to [our blogpost](https://ory.sh/argon2-parameter-choice-best-practice/)
+Head to [our blogpost](https://www.ory.sh/choose-recommended-argon2-parameters-password-hashing/)
```

Comment:
What about docs/docs?

Reply:
You mean `next` ?
There it is already fixed 🤷